### ISSUE TITLE
Website: Enable hooks to individual sections on a page and link back to home

### ISSUE THREAD

**Issue by constantinpape** [mobie/mobie.github.io#86]
These two basic features would be nice to enable better navigation on the website.

Hooks: I mean getting the url to a section of a page, like this:
![image](https://user-images.githubusercontent.com/4263537/189086286-b8b57429-fe6c-457e-b1ad-d7b7c62b6e26.png)

Link to home: I think it would be nice if clicking on the MoBIE logo leads back to the initial web page.

@K-Meech any idea how to do this in the github jekyll theme we're using?

**Comment by K-Meech** [comment]
I agree this would be very useful! I don't think there's an easy way to add those hooks to github pages though unfortunately. They appear automatically for github markdown, but it looks like you'd need to add some extras to make it work for github pages e.g. https://ben.balter.com/2014/03/13/pages-anchor-links/ or https://milanaryal.com.np/adding-hover-anchor-links-to-header-on-github-pages-using-jekyll/ 

For making the image a link, I think this is added here: https://github.com/mobie/mobie.github.io/blob/master/_layouts/default.html#L34 So you probably just need to add a link to this line?
